utils/charts.js

```diff
@@ -13,6 +23,13 @@ export const getPlotSettings = function () {
       'autoScale2d',
       'resetScale2d',
     ],
+    toImageButtonOptions: {
+      filename: plotname,
+      format: 'png',
+      height: 800,
+      width: 1200,
```

Comment:
```suggestion
```

I recommend removing these hard-coded height/width dimensions because it stretches the exported plots in strange ways. I tried removing these two lines on my end and the plots looked great. Probably a good idea to keep the `scale: 2`, though, because high-res plots are great for publications.